src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/BlobManager.java

```diff
@@ -204,7 +206,19 @@ private void moveFileToRejectedContainer(
             ? AccessCondition.generateLeaseCondition(leaseId)
             : AccessCondition.generateEmptyCondition();
 
-        inputBlob.deleteIfExists(DeleteSnapshotsOption.NONE, deleteCondition, null, null);
+        try {
+            inputBlob.deleteIfExists(DeleteSnapshotsOption.NONE, deleteCondition, null, null);
+        } catch (StorageException e) {
+            //if lease lost retry
+            log.info("Deleting File {} got error ", fileName, e);
```

Comment:
perhaps a warning?

Reply:
after this it can still create exception. may be better to make other one WARN which is inside the if ?